source/css/scss/app/loancards-2.scss

```diff
@@ -5,6 +5,28 @@ $loan-card-footer-height: 3rem;
 $loan-card-meter-height: rem-calc(8);
 $loan-card-border: 1px solid $kiva-stroke-gray;
 
+.list-view .list-view-header.is-loan-card-2 {
+	@media #{$xlarge-up} {
+		display: table;
+		width: 100%;
+		padding: 0.5rem 1rem;
+		background-color: $kiva-bg-darkgray;
+		border-bottom: $loan-card-border;
+
+		.loan-country {
+			width: 45.5%;
+		}
+
+		.partner-trustee {
+			width: calc(28.5% - 6.25rem);
+		}
```

Comment:
Nice work here, I'm curious if you had to figure out the right spacing calculation for this or if these css values were handed down to you within the ticket?

Reply:
@BoulderBrains - The general principle was established by Shua for the last implementation of list view - but I had to refine the actual % values in the calc by trial and error. The 6.25rem in the calculation is the width of the picture in the far-left column - and the calc values attempt to size the middle column widths proportionally for screen widths between 761 and 990 px. The far-right column is also sized proportionally but has a min-width so it never gets too small.